layers/+emacs/better-defaults/packages.el

```diff
@@ -10,9 +10,19 @@
 ;;; License: GPLv3
 
 (defconst better-defaults-packages
-  '(mwim)
+  '((ansi-colors :location built-in)
+    mwim)
   "The list of Lisp packages required by the mwim layer.")
 
+(defun better-defaults/init-ansi-colors ()
+  :init
+  (progn
+    (defun better-defaults/compilation-buffer-apply-ansi-colors ()
+      (setq inhibit-read-only t)
+      (ansi-color-apply-on-region compilation-filter-start (point-max))
+      (setq inhibit-read-only nil))
```

Comment:
Use a let binding for `inhibit-read-only` instead of globally modifying the variable with two `setq`, example here: https://github.com/syl20bnr/spacemacs/blob/develop/layers/+tools/shell/packages.el#L137-L138